app/pages/admin/project-status.jsx

```diff
@@ -52,6 +55,13 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  getFeaturedProject() {
+    return apiClient.type('projects').get({ featured: true })
+      .then((featuredProject) => {
```

Comment:
This should be either `(featuredProjects)` to make it clear this is an array of projects, or `([featuredProject])` to get the first project from the array.

Reply:
If I `setState` after getting the project with `([featuredProject])`, then the `state.featured` is always a project. So if I understand correctly, your other comment becomes irrelevant as long as I do `([featuredProject])`. I've tested this change and the function doesn't crash.

Reply:
Yeah, if you’re consistent about it being a project then the comment about arrays is irrelevant.